### PR TITLE
Disable thrift events being sent to acquisitions stream

### PR DESCRIPTION
**Pull request by TomPretty**
## What are you doing in this PR?
Disable thrift events being sent to acquisitions stream. We are updating this stream to consume json so for now we are stopping sending thrift. We've deleted the `KinesisService` from `acquisitions-event-producer` project as the project is very `thrift`y. We will reintroduce a new json kinesis service (e.g in `support-modules`) in a future PR.

[**Trello Card**](https://trello.com/c/Eq7ap8Fc/2690-super-mode-migrate-realtime-acquisitions-stream)

### CODE REVIEW

File: acquisition-event-producer/src/main/scala/com/gu/acquisition/services/DefaultAcquisitionService.scala [hunk 1]
```diff
@@ -27,12 +27,10 @@ class DefaultAcquisitionService(config: DefaultAcquisitionServiceConfig)(implici
   override def submit[A: AcquisitionSubmissionBuilder](a: A)(implicit ec: ExecutionContext) = {
     val ov = ophanService.submit(a).value
     val gv = gAService.submit(a).value
-    val kv = kinesisService.submit(a).value
```
Comment (tomrf1) [inline acquisition-event-producer/src/main/scala/com/gu/acquisition/services/DefaultAcquisitionService.scala:30]: We can go further and actually remove the kinesisService from the `acquisition-event-producer`.
This module is tied to the thrift model. We can later re-implement the kinesis writing logic elsewhere

Comment (TomPretty) [reply acquisition-event-producer/src/main/scala/com/gu/acquisition/services/DefaultAcquisitionService.scala:30]: Okay - I've removed that now!

File: support-modules/acquisitions-stream/src/main/scala/com/gu/support/acquisitions/AcquisitionsStreamService.scala [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    val promise = Promise[Either[String, Unit]]()
+
+    val request: PutRecordRequest = {
+      val buffer = ByteBuffer.wrap {
+        acquisition.asJson.noSpaces.getBytes
+      }
+
+      new PutRecordRequest()
+        .withStreamName(config.streamName)
+        .withPartitionKey(acquisition.identityId.getOrElse(acquisition.amount.toString))
```
Comment (TomPretty) [inline support-modules/acquisitions-stream/src/main/scala/com/gu/support/acquisitions/AcquisitionsStreamService.scala:44]: `amount` here is also an option, so I guess we shouldn't just use `.toString` on it here?

Comment (tomrf1) [reply support-modules/acquisitions-stream/src/main/scala/com/gu/support/acquisitions/AcquisitionsStreamService.scala:44]: Good spot.
The partition key is not especially important here (see [here](https://docs.aws.amazon.com/streams/latest/dev/key-concepts.html) for explanation) as the stream is low throughput.
We could just stringify the `eventTimeStamp` here instead?

Comment (TomPretty) [reply support-modules/acquisitions-stream/src/main/scala/com/gu/support/acquisitions/AcquisitionsStreamService.scala:44]: Yeah sure thing, so we're essentially going to be putting each event into its own shard?

Comment (tomrf1) [reply support-modules/acquisitions-stream/src/main/scala/com/gu/support/acquisitions/AcquisitionsStreamService.scala]: the stream's cloudformation actually defines a `ShardCount` of 1, so whatever we put here will make no difference! If there were more shards then the partition key would determine which one kinesis picks

**Review by tomrf1 — APPROVED** [review]

File: support-modules/acquisitions-stream/src/main/scala/com/gu/support/acquisitions/AcquisitionsStreamService.scala (hunk 2 above)
Comment (TomPretty) [reply support-modules/acquisitions-stream/src/main/scala/com/gu/support/acquisitions/AcquisitionsStreamService.scala]: ahhh gotcha!

**Comment by tomrf1** [comment]
👍 Looks great! We can begin using the new kinesis service in a follow-up PR

**Pull request merged**